src/site/content/en/lighthouse-accessibility/audio-caption/index.md

```diff
@@ -43,7 +57,15 @@ with attribute `kind="captions"`:
 </audio>
 ```
 
+{% Aside %}
```

Comment:
This information is (somewhat) duplicated in the track-kinds include

Reply:
That's true, but they were different enough that I thought it made sense to break them out. For one thing, audio descriptions aren't applicable to `<audio>` elements.

Reply:
ok sounds good